src/data/pages/tos.de.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Nutzungsbedingungen und Datenschutzerklärung
+slug: tos
+---
+
+Datenschutz ist uns ein wichtiges Anliegen und wir bearbeiten deine Daten nur gemäss den gesetzlichen Bestimmungen. In diesen Nutzungsbedingungen erläutern wir, wie wir mit Daten umgehen und welche Rechte du besitzt.
+
+## Annahme der Nutzungsbedingungen
+Durch den Zugriff auf diese Applikation sowie deren Nutzung erklärst du dich mit den vorliegenden Nutzungsbedingungen einverstanden und erkennst diese als verbindlich an. Wenn du mit den Nutzungsbedingungen nicht einverstanden bist, solltest du nicht auf _eCamp_ zugreifen und deren Inhalte und Funktionen nicht nutzen.
+
+## Begriffe
+### _ecamp3_, _eCamp_ und &ldquo;die Applikation&rdquo;
+Bei _ecamp3_ handelt es sich um Open Source Software (siehe Begriff unten) welche die Planung von J+S-Lagern, J+S-Kursen und anderen Lagern vereinfachen soll.
+Der Programmcode der Software _ecamp3_ ist öffentlich auf GitHub unter [https://github.com/ecamp/ecamp3](https://github.com/ecamp/ecamp3){target=_blank} verfügbar und unterliegt der &ldquo;GNU Affero General Public License v3.0&rdquo; Lizenz, siehe Begriff unten.
+
+_eCamp_ ist der Name der öffentlichen Instanz von _ecamp3_, welche wir betreiben, und wird im Folgenden auch &ldquo;die Applikation&rdquo; genannt. Erreichbar ist die Applikation unter der URL [https://ecamp3.ch](https://ecamp3.ch/){target=_blank}.
+
+### Nutzende
+Nutzende sind natürliche Personen wie du, welche die Planungsfeatures von _eCamp_ nutzen und Inhalte auf _eCamp_ führen möchten. Wer _eCamp_ nutzen und Inhalte erfassen möchte, benötigt hierzu zwingend ein Nutzenden-Konto.
+
+### Nutzenden-Konto
+Ein Nutzenden-Konto erlaubt es, Lager zu erstellen und an Lagern mitzuarbeiten. Zur Erstellung eines Nutzenden-Kontos werden im Minimum ein zweiteiliger Name, eine E-Mail-Adresse und ein Passwort benötigt. Diese Informationen sind für alle anderen Nutzenden verfügbar, welche mit dir an Lagern zusammenarbeiten.
```

Comment:
Zweiteilig ist nicht mandatory, oder? Passwort ist nicht zwingend notwendig (OAuth) + ist nicht kompatibel mit dem nächsten Satz "für alle anderen Nutzenden verfügbar"

```suggestion
Ein Nutzenden-Konto erlaubt es, Lager zu erstellen und an Lagern mitzuarbeiten. Zur Erstellung eines Nutzenden-Kontos werden im Minimum ein Name und eine E-Mail-Adresse benötigt. Diese Informationen sind für alle anderen Nutzenden verfügbar, welche mit dir an Lagern zusammenarbeiten.
```

Reply:
Zweiteilig ist eben aktuell im Frontend mandatory habe ich gemerkt:
![Screenshot 2022-12-05 at 16-32-44 eCamp3](https://user-images.githubusercontent.com/7566995/205677005-1e2f2232-8c41-4f17-a8dd-fd3418823107.png)

In der API ist keines von beidem required. Sollten wir vielleicht noch ändern, und dann die TOS anpassen.

Aber ja, muss mal überlegen wie wir OAuth hier berücksichtigen könnten.

Reply:
Habe mal https://github.com/ecamp/ecamp3/issues/3196 eröffnet. Schreibe hier aber noch etwas um damits für OAuth und nicht-öffentliche Felder auch passt. Merci für den Hinweis!